config/rbac/role.yaml

```diff
@@ -90,6 +90,23 @@ rules:
       - patch
       - update
       - watch
+  ##
+  ## Rules for tower.ansible.com/v1alpha1, Kind: AnsibleWorkflow
+  ##
+  - apiGroups:
+      - tower.ansible.com
+    resources:
```

Comment:
We can combine these 3 resources with the resources section above (they have the same rules as far as I can tell).

Reply:
resolved in #94